.travis.yml

```diff
@@ -28,15 +24,11 @@ install:
 
 script:
   - make lint
-  - if [ $INTEGRATION == 'true' ]; then
-      make build || travis_terminate 1;
-      ./test-scripts/cluster-up-minikube.sh  || travis_terminate 1;
-      make integration-tests || travis_terminate 1;
-    else
-      make test || travis_terminate 1;
-      make build || travis_terminate 1;
-      make coverage || travis_terminate 1;
-    fi
+  - make test
+  - make build
+  - ./test-scripts/cluster-up-minikube.sh
```

Comment:
This is more of an install step I would think. It's not part of the actual testing.